pkgs/os-specific/linux/rust-out-of-tree-module/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, kernel }:
+kernel.stdenv.mkDerivation {
```

Comment:
Why not normal stdenv?

Reply:
Kernel modules should be built with the stdenv that the kernel itself was built with. Otherwise, it might lead to weird problems. I think we're not very consistent about this though.